bundles/org.openhab.binding.robonect/src/main/java/org/openhab/binding/robonect/internal/model/Blades.java

```diff
@@ -0,0 +1,48 @@
+/**
+ * Copyright (c) 2010-2023 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.robonect.internal.model;
+
+/**
+ * Blade information from the mower.
+ *
+ * @author Christian Jonak-Moechel
+ */
+public class Blades {
```

Comment:
This class could be annotated to avoid SAT warning "Classes/Interfaces/Enums should be annotated with @NonNullByDefault".
```suggestion
@NonNullByDefault
public class Blades {
```